lib/guard/puma/runner.rb

```diff
@@ -69,6 +71,24 @@ def sleep_time
 
     private
 
+    PUMA_OPTIONS_KEYS_BY_PUMACTL = {
+      true => {
+        config:      '--config-file',
+        control_url: '--control-url'
+      }.freeze,
```

Comment:
These inner hashes don't also need to be frozen. The outer hash being frozen freezes them. :snowflake:

Reply:
No: https://repl.it/repls/EsteemedSubstantialPoints

Reply:
Weird, I must have mistyped something in a previous IRB session.

I don't think it's really necessay to deep freeze the whole thing anyway, at least not right now (given no real issues on it)